sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -120,15 +125,44 @@ class KubernetesCmdRunner(RemoteCmdRunnerBase):
     exception_retryable = (ConnectionError, MaxRetryError, ThreadException)
     default_run_retry = 8
 
-    def __init__(self, kluster, pod_name: str, container: Optional[str] = None,
+    def __init__(self, kluster, pod_image: str,  # pylint: disable=too-many-arguments
+                 pod_name: str, container: Optional[str] = None,
                  namespace: str = "default") -> None:
         self.kluster = kluster
+        self.pod_image = pod_image
         self.pod_name = pod_name
         self.container = container
         self.namespace = namespace
 
         super().__init__(hostname=f"{pod_name}/{container}")
 
+        # NOTE: create also the dynamic loader runner for fast utility calls of other loader types
+        self.dynamic_remoter = KubernetesPodRunner(
+            kluster=self.kluster,
+            template_path=sct_abs_path("sdcm/k8s_configs/loaders/pod.yaml"),
+            template_modifiers=list(self.kluster.calculated_loader_affinity_modifiers),
+            pod_name_template=f"{self.pod_name}-dynamic-pod",
+            namespace=self.namespace,
+            environ={
+                "K8S_NAMESPACE": self.namespace,
+                "K8S_LOADER_CLUSTER_NAME": self.pod_name.rsplit("-", 1)[0],
+                "K8S_LOADER_NAME": self.pod_name,
+                # TODO: place some values for below env vars when those become required
+                #       in the dynamic loader config.
+                # "POD_CPU_LIMIT": "foo",
+                # "POD_MEMORY_LIMIT": "bar",
+            },
+        )
+
+    def run(self, cmd, **kwargs):  # pylint: disable=arguments-differ
+        if is_scylla_bench_command(cmd) and hasattr(self, "dynamic_remoter") and hasattr(
```

Comment:
Some loader specific (or even more detailed: scylla-bench) code is leaking to `KubernetesCmdRunner` class which is quite generic.
We should avoid patching generic classes with low level details. Better make `BasePodContainer` code to create proper remoter class instead patching here.

Reply:
I disagree. The thing you are proposing to implement requires lots of changes in places where we call this code.
The solution proposed here is one-place change, it is small and solves the problem.

Reply:
I hoping it does not require lot's of changes - `KubernetesCmdRunner` changed required input parameters and there's only one place use was updated. 
If it's not like I think, I understand that in this case it requires more refactor and you decided to go simple way.

Reply:
It is not about initialization. It is about calling the `node.remoter.run(...)` method.
After implementation of your proposal, it will be different method, for example `node.remoter.run_dynamic(...)`.
And, it may be called `interface change`, and, obviously, it requires changes in lots of places.